src/api/java/mekanism/api/gear/ModuleData.java

```diff
@@ -275,4 +314,61 @@ public ModuleDataBuilder<MODULE> disabledByDefault() {
             return this;
         }
     }
+
+    /**
+     * Enum of flags for module exclusivity channels
+     */
+    public enum ExclusiveFlag {
+
+        /**
+         * This flag indicates that this module uses interaction without a target
+         */
+        INTERACT_EMPTY,
+
+        /**
+         * This flag indicates that this module uses interaction with an entity
+         */
+        INTERACT_ENTITY,
+
+        /**
+         * This flag indicates that this module uses interaction with a block
+         */
+        INTERACT_BLOCK,
+
+        /**
+         * This flag indicates that this module changes what pressing jump does
+         */
+        OVERRIDE_JUMP;
+
+        // This will work for 32 flags, if we go past that we will have to switch it to a long. But that is a lot of flags!
+
```

Comment:
Spacing around this comment is a bit odd, feels like it got somewhat lost due to it not being a comment on anything in particular

Reply:
I just got rid of it, most people know sizes I think if it becomes a problem which it most likely won't